lib/plugins/wallet/bitcoind/bitcoind.js

```diff
@@ -36,15 +36,15 @@ function checkCryptoCode (cryptoCode) {
 
 function accountBalance (cryptoCode) {
   return checkCryptoCode(cryptoCode)
-    .then(() => fetch('getwalletinfo'))
-    .then(({ balance }) => new BN(balance).shiftedBy(unitScale).decimalPlaces(0))
+    .then(() => fetch('getbalances'))
+    .then(({ mine }) => new BN(mine.trusted).shiftedBy(unitScale).decimalPlaces(0))
     .catch(errorHandle)
 }
 
 function accountUnconfirmedBalance (cryptoCode) {
   return checkCryptoCode(cryptoCode)
-    .then(() => fetch('getwalletinfo'))
-    .then(({ unconfirmed_balance: balance }) => new BN(balance).shiftedBy(unitScale).decimalPlaces(0))
+    .then(() => fetch('getbalances'))
+    .then(({ mine }) => new BN(mine.untrusted_pending).plus(mine.immature).shiftedBy(unitScale).decimalPlaces(0))
```

Comment:
Are we sure we want to add the immature here?

Reply:
I believe this was overlooked on our end. Despite no one actually using the nodes for mining if, in the future and for some reason, some operator decides to use a node to operate as the lamassu node and a mining node at the same time, our pending balance was not taking into account balance from coinbase rewards (which is the immature balance there). I can chat about it with Rafael to get a stance on this